src/views/Activity.vue

```diff
@@ -42,14 +40,22 @@ div
 
   hr
 
-  p Showing activity from {{ date }} until 24 hours later
-
-  p Events queried: {{ eventcount }}
+  p
+    //| Showing activity from {{ date }} until 24 hours later
+    //br
+    | Events queried: {{ eventcount }}
 
 </template>
 
 <style lang="scss">
 
+#apptimeline-container {
+    white-space: nowrap;
+    font-family: arial;
+    font-size: 10pt;
```

Comment:
This is a bit too small in my opinion. I personally like small fonts, but we should consider others with slightly worse than average vision and have 11 or 12pt as a minimum everywhere.